cmd/rollouts-controller/main.go

```diff
@@ -69,6 +69,7 @@ func newCommand() *cobra.Command {
 		awsVerifyTargetGroup bool
 		namespaced           bool
 		printVersion         bool
+		prometheusAddress    string
```

Comment:
We should configure this via configmap/secret instead of CLI flag (just like we configure datadog, new relic, wavefront). Could we switch to that model instead?

Reply:
Hmmm. What do you think about configuring this using an environment variable instead? ConfigMap or Secret doesn't look like a good place to configure this. Maybe I'm missing some background/context.

Reply:
Looking a some k8s references: https://kubernetes.io/docs/reference/command-line-tools-reference I thought cli paramter option was good. I can update with configmap / secret.. however would like to understand why they are preferred?